tests/tests.c

```diff
@@ -50,9 +50,9 @@
 
 test_pattern_t test_list[NUM_TEST_PATTERNS] = {
     // ena,  cpu, stgs, itrs, errs, description
-    { true,  SEQ,    1,    6,    0, "[Address test, walking ones, no cache] "},
+    { true,  ONE,    1,    6,    0, "[Address test, walking ones, no cache] "},
     {false,  SEQ,    1,    6,    0, "[Address test, own address in window]  "},
```

Comment:
ONE for test 1 as well ? It's disabled by default, but it should probably have the same running mode as test 1.